Kernel/Net/E1000ENetworkAdapter.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/MACAddress.h>
+#include <Kernel/Debug.h>
+#include <Kernel/Net/E1000ENetworkAdapter.h>
+#include <Kernel/PCI/IDs.h>
+
+namespace Kernel {
+
+#define REG_EEPROM 0x0014
```

Comment:
(there's no need to address this, it's just a rant)
\<rant>
What has `constexpr auto` done to you kernel guys? why is the kernel so full of `#define`'s? :cry:
\</rant>

Reply:
https://en.wiktionary.org/wiki/there%27s_no_accounting_for_taste
I really like the translation of this phrase in Hebrew :)

Reply:
If I was the first to write the networking code, I think the better choice is to use `enum`s and `enum class`es. We should be consistent and if needed, we can change the style altogether, but it might be worse to leave half the code in one style, the other half with another style.

Reply:
It's alright to keep it as is for now, if we decide to de-`#define`-ify the kernel some day, that will get its own PR :P